jumpscale/packages/notebooks/package.toml

```diff
@@ -9,3 +9,5 @@ port = 8888
 path_url = "/notebooks"
 path_dest = "/notebooks"
 websocket = true
+nginx_proxy_buffers = "8 256k"
```

Comment:
I think `proxy_buffers` is enough? (without nginx)
Also, add buffering enabled field too.